### ISSUE TITLE
Firefox 33.0.1 and FireGestures 1.8.6 constantly pulling up the context menu upon right click

### ISSUE THREAD

**Issue by nwilhelm** [gomita/firegestures#90]
Almost every time I try to begin a gesture I end up pulling down the context menu. It is something I have only noticed in the past few days. I haven't personally changed any settings but I know FF updated from 32 to 33.0.1

**Comment by mehturt** [comment]
Yes, so this does not work for me as well, running FF nightly (38.0)

**Comment by Mithgol** [comment]
Then this issue might be related to #91 (though in the latter “Some bugs related to mouse gesture with left-button and middle-button were fixed” but here it's an issue caused by a right click).

**Comment by mehturt** [comment]
Yes, that's what I meant, #91. Because it stopped working for me ever since FF started to experiment with e10s.

**Comment by gargeug** [comment]
I am having this issue right now and it is very annoying. I am on Firefox 37.0.1. I noticed it when I upgraded Firefox from 35 (I think) to 37.0.1. I was using the current release of FG. I updated to the newest beta 1.9b5 to see if it was fixed, but it is still occurring. Every time I right click to do a gesture, the context menu pops up. I can still perform the gesture, but only if I hold the right click down for a second before moving the cursor. If I right click and start moving instantly, FG will interpret the move as complete after about a second and perform the action of what I was able to complete in that second before I even finish the move or let go of the right click. The context menu always pops up, and remains popped up throughout all of this. Is anyone else having similar issues still?

**Comment by ush189** [comment]
Same here on FF 38.0.5

**Comment by paskalito** [comment]
samo for me since like forever, and since i month im tryin to find ways to fix it.., but actually the only thing that makes it work is using middle (scroll wheel) / since if you use the left button you cannot drag and drop anymore... -_-. - i also deactivated all other addons but its still the same.
im on win7 currenty ff 39.0

**Comment by kkurola** [comment]
Same here. For example, I have "close tab" set as up&down gesture holding mouse2, but every now and then the gesture is trickered while I'm still going up resulting in up gesture, which I've set to open a new tab. The gesture trail still keeps drawing after the false gesture is trickered. I'm not sure if I could do another gesture afterwards as I usually draw the gesture too fast to react and test things out. Sometimes the new tab is immediately closed afterwards, so it's possible I accidentally complete a new up&down gesture after the new tab gesture is trickered.

In my case this behaviour started sometime around July or early August. Now using FG 1.10.1 with Firefox 40.0.3
